src/components/UserProfile/UserProfile.jsx

```diff
@@ -290,6 +290,7 @@ const UserProfile = props => {
     }
     try {
       await props.updateUserProfile(props.match.params.userId, userProfile);
+      props.refreshToken(userProfile._id);
```

Comment:
Have you considered making this await ? Thanks.

Reply:
@RJBornToWin Thank you! I forgot to making this await, I'll change it.

Reply:
@RJBornToWin  and @rauleffting  can you guys review again? I added an `await` that I forgot to add before.

Reply:
I tested again. It worked perfectly! Thank you, Johny!